src/css/mediaelementplayer-legacy.css

```diff
@@ -1,7 +1,6 @@
 /* Accessibility: hide screen reader texts (and prefer "top" for RTL languages).
 Reference: http://blog.rrwd.nl/2015/04/04/the-screen-reader-text-class-why-and-how/ */
 .mejs-offscreen {
-    clip: rect(1px, 1px, 1px, 1px); /* IE8-IE11 - no support for clip-path */
     clip-path: polygon(0 0, 0 0, 0 0, 0 0);
```

Comment:
Update to latest version of http://blog.rrwd.nl/2015/04/04/the-screen-reader-text-class-why-and-how/